### ISSUE TITLE
Qt6 migration

### ISSUE THREAD

**Issue by JamiKettunen** [maltejur/discord-screenaudio#117]
I believe the Qt6 WebEngine will also be based on a much newer build of Chromium and could fix a bunch of the random crashes etc currently plaguing this app

**Comment by maltejur** [comment]
Since I use some of the KDE Frameworks, this probably still has to wait until KDE has officially switched to Qt6.

**Comment by maltejur** [comment]
Actually, it may be a good idea to update the CMake config right now to allow easier building with Qt6, Flatpak for example already offers a Qt6 KDE Runtime.

**Comment by sektr63a** [comment]
@maltejur sorry fo my (probably stupid) question, I notice there KDE in use. So if I use Gnome de, will it work for me?

**Comment by JamiKettunen** [comment]
@sektr63a Obviously, you can even run all KDE software on GNOME or any desktop environment / window manager for that matter, just it may look a bit out of place or have some missing integration with the rest of the desktop environment
